pshtt/cli.py

```diff
@@ -56,7 +57,8 @@ def main():
         'timeout': args['--timeout'],
         'preload_cache': args['--preload-cache'],
         'suffix_cache': args['--suffix-cache'],
-        'cache': args['--cache']
+        'cache': args['--cache'],
+        'ca_file': args['--ca-file']
```

Comment:
As a minor not, if you add the comma at the end here you won't have to modify this line when you add the next line.